functions/main.py

```diff
@@ -0,0 +1,150 @@
+# Welcome to Cloud Functions for Firebase for Python!
+# To get started, simply uncomment the below code or create your own.
+# Deploy with `firebase deploy`
```

Comment:
Nice! This will save the effort of moving the back-end and the front-end into the same repo <3

Reply:
<3 plus emulators support.